lib/menu.rb

```diff
@@ -0,0 +1,49 @@
+require 'twilio-ruby'
+
+account_sid = "ACef87d7dfc16f3cebd55fc9ec9969335a"
+
+class Menu
+
+    attr_reader :menu, :order, :order_total
+
+    def initialize
+        @menu = { "hamburger" => 2, "pizza" => 5 }
+        @order = []
+        @order_total = []
+    end
+
+    def check_menu
+        menu
+    end
+
+    def select_dishes(*selection, total)
+        selection.each do |i|
+            if menu.has_key?(i)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.